rust/Cargo.toml

```diff
@@ -29,6 +29,7 @@ serde = { version = "1.0", features = ["derive"] }
 serde_yaml = "0.9"
 unindent = "0.2.4"
 encoding_rs = "0.8.35"
+once_cell = "1.21.3"
```

Comment:
Do we need to add `once_cell`? Can we use `lazy_static` (already a project dependency) or even `std::cell::OnceCell`?

Reply:
Good idea - I've switched to `lazy_static` for consistency.